providers/bind/bindProvider.go

```diff
@@ -44,6 +44,7 @@ var features = providers.DocumentationNotes{
 	providers.CanUseSRV:              providers.Can(),
 	providers.CanUseSSHFP:            providers.Can(),
 	providers.CanUseTLSA:             providers.Can(),
+	providers.CanUseDHCID:            providers.Can(),
```

Comment:
Please keep this list alphabetized.   (`CanUseDHCID` should be after `CanUseCAA`)